rasa/utils/tensorflow/exceptions.py

```diff
@@ -3,3 +3,7 @@
 
 class TFLayerConfigException(RasaException):
     """Raised when wrong parameters are passed to tensorflow layers."""
+
+
+class TFModelConfigException(RasaException):
```

Comment:
I am terribly sorry but do you think it makes sense to rename this exception type to `RasaModelConfigException`. All modes inside Rasa do inherit from a base `RasaModel`. No point adding `TF` in front.

Reply:
No worries, yep that's much nicer :)